src/deprecation.hpp

```diff
@@ -21,7 +21,8 @@ enum class DEP_LEVEL : uint8_t { INDEFINITE = 1, PREEMPTIVE, FOR_REMOVAL, REMOVE
 
 /**
  * Prints a message to the deprecation log domain informing players that a given feature
- * has been deprecated.
+ * has been deprecated. May show the error in the in-game chat area, but has a mechanism
+ * to avoid repeatedly showing in-game messages to players who can't fix the issue.
```

Comment:
I need to check this statement - I thought it was true, but I don't see the mechanism here, maybe it's only implemented in the Lua layers. That would explain the large number of repeated deprecation warnings about the SPECIAL_NOTES macro.

Reply:
I think the mechanism in question is part of `lg::wml_error`.

Reply:
Excellent, that's definitely being used hee.